libr/core/cconfig.c

```diff
@@ -3971,7 +3966,13 @@ R_API int r_core_config_init(RCore *core) {
 	SETI ("lines.from", 0, "Start address for line seek");
 	SETCB ("lines.to", "$s", &cb_linesto, "End address for line seek");
 	SETCB ("lines.abs", "false", &cb_linesabs, "Enable absolute line numbers");
-
+	/* RVC */
+	{
+		char *p = r_file_path ("git");
+		bool found = (p && (*p == '/' || *p == 'g'));
```

Comment:
the correct conditional should be like this:

```suggestion
		bool found = (p && *p != 'g');
```

Reply:
I forgot to git pull :(